config-connector/solutions/sql/kpt/mysql-public/Kptfile

```diff
@@ -0,0 +1,36 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: mysql-public
+packageMetadata:
+  shortDescription: a public mySQL cluster
```

Comment:
`configures` a public `MySQL` cluster.
Let's use a short sentence in the description, so that it will be consistent with the other solutions.
Also `MySQL` is a special term usually starts with a capital M.